src/mumble/Database.cpp

```diff
@@ -165,16 +178,22 @@ Database::Database() {
 	execQueryAndLogFailure(query, QLatin1String("SELECT sqlite_version()"));
 	while (query.next())
 		qWarning() << "Database SQLite:" << query.value(0).toString();
+
+	return db;
 }
 
-Database::~Database() {
-	QSqlQuery query;
+void Database::cleanupDatabase() {
```

Comment:
The cleanup was previously executed on destruction of `Database`. This is now missing.

Reply:
`main()` is now responsible for closing all the connections.

Reply:
Why is that a separate commit? This commit makes no sense without the other one, right?

Reply:
This method only closes a single connection; the one for the thread that calls this method.

Reply:
Indeed, I'm going to merge them.

Reply:
> This method only closes a single connection; the one for the thread that calls this method.
So main will not clean up all connections, but only its own.

A thread can close a connection even if it's created by another one.

Reply:
Ah, I missed the loop in main